app/views/courses/StudentRankingView.coffee

```diff
@@ -0,0 +1,198 @@
+    @studentsID.push('5cd42e36c75fba003f06e933')
+    @sessions.push(@supermodel.loadCollection(new LevelSessionsCollection(LevelSession,'5cd42c5615f63b002dd42192'), 'your_sessions', {cache: false}, 1).model)#Francisco
+    @studentsID.push('5cd42c5615f63b002dd42192')
+    @sessions.push(@supermodel.loadCollection(new LevelSessionsCollection(LevelSession,'5cd42bc615f63b002dd41ec5'), 'your_sessions', {cache: false}, 1).model)#João
+    @studentsID.push('5cd42bc615f63b002dd41ec5')
+    @sessions.push(@supermodel.loadCollection(new LevelSessionsCollection(LevelSession,'5cd4353956eba9004c82af45'), 'your_sessions', {cache: false}, 1).model)#Kayllane
+    @studentsID.push('5cd4353956eba9004c82af45')
+    @sessions.push(@supermodel.loadCollection(new LevelSessionsCollection(LevelSession,'5cd429a39df7ff002b3b9c03'), 'your_sessions', {cache: false}, 1).model)#Khawe
+    @studentsID.push('5cd429a39df7ff002b3b9c03')
+    @sessions.push(@supermodel.loadCollection(new LevelSessionsCollection(LevelSession,'5cd44e8b15f63b002dd4b01a'), 'your_sessions', {cache: false}, 1).model)#Kildere
+    @studentsID.push('5cd44e8b15f63b002dd4b01a')
+    @sessions.push(@supermodel.loadCollection(new LevelSessionsCollection(LevelSession,'5cd42c298797f30022639473'), 'your_sessions', {cache: false}, 1).model)#Livia
+    @studentsID.push('5cd42c298797f30022639473')
+    @sessions.push(@supermodel.loadCollection(new LevelSessionsCollection(LevelSession,'5cd42d547610750045a80c0f'), 'your_sessions', {cache: false}, 1).model)#Luis
+    @studentsID.push('5cd42d547610750045a80c0f')
+    @sessions.push(@supermodel.loadCollection(new LevelSessionsCollection(LevelSession,'5cd42d4c1ec72d00279cd9db'), 'your_sessions', {cache: false}, 1).model)#Lydia
+    @studentsID.push('5cd42d4c1ec72d00279cd9db')
+    @sessions.push(@supermodel.loadCollection(new LevelSessionsCollection(LevelSession,'5cd42d7ab9d8db00286a8013'), 'your_sessions', {cache: false}, 1).model)#Vinicius
+    @studentsID.push('5cd42d7ab9d8db00286a8013')
+    @sessions.push(@supermodel.loadCollection(new LevelSessionsCollection(LevelSession,'5cd42bb1db1a5f00420ffa73'), 'your_sessions', {cache: false}, 1).model)#Wenddel
+    @studentsID.push('5cd42bb1db1a5f00420ffa73')
+    @sessions.push(@supermodel.loadCollection(new LevelSessionsCollection(LevelSession,'5cd42ac21ec72d00279cd100'), 'your_sessions', {cache: false}, 1).model)#Wystefani
+    @studentsID.push('5cd42ac21ec72d00279cd100')
+
+    #@sessions.push(@supermodel.loadCollection(new LevelSessionsCollection(LevelSession,'5a217768db5a1a00850eb52d'), 'your_sessions', {cache: false}, 1).model)#TESTE
+    #@studentsID.push('5a217768db5a1a00850eb52d')
+  
+  fetchStudents: () ->
+    Promise.all(@students.fetchForClassroom(@classroom, {removeDeleted: true, data: {project: 'firstName,lastName,name,email,coursePrepaid,coursePrepaidID,deleted'}}))
+    .then =>
```

Comment:
typo?

Reply:
This is necessary to call some of the data of the students, from this it is counted part of the ranking. I have not yet found a different way for Promise to do this. So I'm using Promise there but I have not seen the need to execute anything yet. It is still necessary to improve this section.